src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -19,7 +23,7 @@ public void showIOError() {
     }
 
     public void showNumberFormatError() {
-        System.out.println("The input is not a valid number! Please try again.");
+        System.out.println("either The amount or deadline input is not a valid number! Please try again.");
```

Comment:
Bruh just handle the errors separately, you can use different catch blocks for numberFormatException or DeadlineFormatException. The current error message does not make sense if input is wrong for deposit or withdraw commands